drivers/hmis/app/models/hmis/staff_assignment_relationship.rb

```diff
@@ -5,6 +5,9 @@
 ###
 
 class Hmis::StaffAssignmentRelationship < Hmis::HmisBase
+  # Note that this model isn't currently tied to a data source, which means that if we have multiple HMISes
+  # within the same installation, they will all see the same options for StaffAssignmentRelationship.
+  # We could change this if the need arises.
```

Comment:
Good thought. I have been adding bullet points to https://github.com/open-path/Green-River/issues/6612 as I notice things, recently. The big one is the config tables (https://github.com/open-path/Green-River/issues/6691) and I think this falls into that category. Added a bullet point there